spec/features/user_asks_question_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+describe "ask a question", type: :feature do
+
+  it "displays the asked question" do
+    login_and_ask_question
```

Comment:
At this point, I like having each example separated out.  I tried to put the login_and_ask_question method call in a before(:all) block only to run into a bunch of problems (needed database_cleaner and then realized the Capybara resets the session between examples).  However, I understand your point.  After I'm done refactoring this part of the site, I'll refactor the test suite to have multiple examples in each scenario.